### DISCUSSION TITLE
Create a backup from 1.0 Beta 5

### DISCUSSION THREAD

**Discussion by Sar6e** [mealie-recipes/mealie#4100] [Q&A] [answered]
Hopefully not asking the dumb question.  I am planning out how to move Mealie to a new server and upgrade from v 1.0.0 Beta 5 to the current version.  The plan is to run a backup and then simply import that into the new instance on the new machine.

The bit I'm struggling with is to work out how to create the backup.

I can find doco on how to run a backup from pre v1.0 systems.  None of that matches what I'm seeing.  When I go to profile, I can see a panel for Data Migrations.  In there I can see where I imported from the pre V1.0 system into the current beta 5 system.  What I can't see there is an option to create a backup.  Hopefully I'm not going to be deeply embarrassed by somebody pointing out the bleedingly obvious.

I'm hoping there is not a separate admin account I've forgotten the password for.  The DB only shows the 2 user accounts that I'd expect.

Thanks in anticipation.

**Comment by Sar6e** [comment]
I copied the container data on the current system and stood up a new container running 1.12.  That stood up without incident and I'm seeing different options in the list down the left hand side but still no option to create a backup.

**Comment by Kuchenpirat** [comment]
You can manage backups in the admin setting section and then under backups or at `/admin/backups`

There you can create, upload and restore post v1 backups.

I think you are currently browsing the migration page, which is used to migrate from another platform (or pre v1 mealie)

**Reply by Sar6e** [reply]
Hi @Kuchenpirat (cool username!),

I think that's my problem.  I'm not sure how I navigate to the admin settings.  The navigation panel I'm seeing lists Recipes, Meal Planner, Shopping Lists, Timeline, Cookbooks and an exapndable Organizers section.  This expands out to list Categories, Tags and Tools.  There's no "Settings" available.

In the User Profile page I can see panels for Manage Data and Data Migrations.  

![image](https://github.com/user-attachments/assets/8339c4b4-7f16-4e7d-8cf5-34f4a223340b)

**Reply by Kuchenpirat** [reply, marked as answer]
Thanks 😊

OK, my guess is that your user is no admin.

Does the original user still exist that one should be an admin account.
